__tests__/routes/actions.test.js

```diff
@@ -3,6 +3,7 @@ const express = require('express');
 const actionsModel = require('../../api/actions/actionsModel');
 const actionsRouter = require('../../api/actions/actionsRouter');
 const server = express();
+//You will get two 500 errors, this is not failed tests this was intentional
```

Comment:
Maybe you can make the comment a bit more specific (where do the errors pop up, why are they there, what is throwing the error)

Reply:
I agree a little more detail would really help the next cohort out.

Reply:
I agree, it's not apparently obvious.

Reply:
You could also indicate that they are intentional in the documentation for the next cohort, might help them see it faster? Just a thought.